auth/controllers/userController.ts

```diff
@@ -0,0 +1,79 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import User from "../models/user";
+import { addUserToDB, authUser } from "../services/AuthService";
+import { hashPassword, ReE, ReS, to } from "../services/Util";
+
+export async function loginUser(req: Request, res: Response): Promise<void> {
+  res.setHeader("Content-Type", "application/json");
+
+  const body = req.body;
+  let err;
+  let user: any;
```

Comment:
The user should be an instance of the User class right? It would be good to specify the type for all the variables, since that's the main benefit of using Typescript and it'll make adding to the codebase easier later on.

Reply:
Yeah, you are right, my bad I should've used User !! Thanks 👍